packages/trpc/server/routers/viewer/eventTypes.tsx

```diff
@@ -99,6 +99,14 @@ const EventTypeUpdateInput = _EventTypeModel
       })
   );
 
+const EventTypeDuplicateInput = z.object({
+  id: z.number(),
+  slug: z.string(),
+  title: z.string(),
+  description: z.string(),
+  length: z.number(),
```

Comment:
This is all we need for duplicating and event, form inputs and we can get the rest from DB.